bitnami/prometheus-operator/templates/alertmanager/ingress.yaml

```diff
@@ -0,0 +1,29 @@
+{{- if and .Values.alertmanager.enabled .Values.alertmanager.ingress.enabled }}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ template "prometheus-operator.alertmanager.fullname" . }}
+  labels: {{ include "prometheus-operator.alertmanager.labels" . | nindent 4 }}
```

Comment:
When using `nindent` make sure you add `{{- ` so the rendered yaml does not contain trailing whitespaces